src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerAlleleBySample.java

```diff
@@ -58,7 +59,9 @@ public void annotate(final ReferenceContext ref,
         for ( final Allele allele : vc.getAlleles() ) {
             alleleCounts.put(allele, 0);
         }
-        likelihoods.bestAlleles(g.getSampleName()).stream()
+        final Map<Allele, List<Allele>> alleleSubset = alleles.stream().collect(Collectors.toMap(a -> a, a -> Arrays.asList(a)));
```

Comment:
This is not the code change in this PR, but In #2185 you kept the error message "not a strict subset of _per read allele map alleles_" in DepthPerAlleleBySample and others. It's not a big deal obviously but wanted to bring it up in case it wasn't intentional.